pathway-analysis/qusage_meta_analysis.Rmd

```diff
@@ -188,16 +199,23 @@ We need to unzip it here if we have not already.
 
 ```{r}
 # if the unzipped folder does not exist
-if (!dir.exists(file.path("data", "GSE37382"))) {
-  compressed_file <- file.path("data", "GSE37382.zip")
-  unzip(compressed_file, exdir = "data")
+if (!dir.exists(file.path(data_dir,
+                          "GSE37382" # Replace with the name of your file without the .zip extension
+                          ))) {
+  compressed_file <- file.path(data_dir,
```

Comment:
We should probably provide a bit more context around this zip file. I think its unlikely the user will have a zip file, but we may want to explain they should skip/delete this step if they don't have a zip file (which I don't think they will).